source/operations/cert-manager.rst

```diff
@@ -0,0 +1,577 @@
+At the namespace level, cert-manager issues certificates derived from an ``Issuer``.
+A ``ClusterIssuer`` can issue certificates for multiple namespaces.
+An ``Issuer`` only issues certificates for its own namespace.
+
+To learn more about cert-manager Issuers, refer to the `Issuer documentation <https://cert-manager.io/docs/concepts/issuer/>`__.
+
+Below is a logical view of a Kubernetes cluster with four namespaces:
+
+- ``minio-operator``
+- ``tenant-1``
+- ``tenant-2``
+- ``other-namespace``
+
+.. image:: /images/k8s/cert-manager-namespaces.svg
+   :width: 600px
+   :alt: A graphic depiction of a kubernetes cluster with four namespaces. Each namespace in its own white box. One is labeled minio-operator with contents two sts content items. A second is labeled tenant-1 with a MinIO logo and four drives of a pool. A third is labeled tenant-2 with similar contents to the tenant-1 box. The fourth is labeled other namespace with contents of other pods.
+   :align: center
+
+This guide shows you how to set up a different Certificate Authority (CA) in each namespace.
+Each namespace's CA references the global ``Cluster Issuer``.
+
+
+The following graphic depicts how various namespaces make use of either an ``Issuer`` or ``ClusterIssuer`` type.
+
+- cert-manager is installed in the ``cert-manager`` namespace, which does not have either an ``issuer`` or a ``ClusterIssuer``.
+- The ``default`` namespace receives the global ``Cluster Issuer``.
+- Each tenant's namespace receives a local ``Issuer``.
+- The ``minio-operator`` namespace receives a local ``Issuer``. 
+  More about services that require TLS certificates in the ``minio-operator`` namespace are covered :ref:`below <minio-operator-services-with-cert-manager>`.
+
+.. image:: /images/k8s/cert-manager-cluster.svg
+   :width: 600px
+   :alt: A graphic depiction of a kubernetes cluster with five separate namespaces represented by different boxes. One is labeled minio-operator with a text box that says "minio-operator: issuer". A second is labeled tenant-1 with text box that says "tenant-1: issuer". A third is labeled default with a text box that says "root:ClusterIssuer". A fourth is labeled tenant-2 with a text box that says "tenant-2: issuer". The fifth is labeled cert-manager with a text box that says "minio-operator: issuer".
+   :align: center
+
+.. note::
+
+   This guide uses a self-signed ``Cluster Issuer``. 
+   You can also use `other Issuers supported by cert-manager <https://cert-manager.io/docs/configuration/issuers/>`__.
+   The main difference is that you must provide the ``Issuer`` CA certificate to MinIO, instead of the CA's mentioned in this guide.
+
+
+Prerequisites
+-------------
+
+- A `supported version of Kubernetes <https://kubernetes.io/releases/>`__. 
+ 
+  While cert-manager supports `earlier K8s versions <https://cert-manager.io/docs/installation/supported-releases/>`__, the MinIO Operator requires a currently supported version.
```

Comment:
I'd drop this. We state what the supported version is, that's all we need.

Reply:
Dropped.